aurora/main.f90

```diff
@@ -149,22 +160,26 @@ subroutine run(  &
   !     rn_out       real*8 (ir,nion,nt_out)
   !                    Impurity densities (temporarily) in the magnetically-confined plasma at the
   !                    requested times [1/cm^3].
-  !     N_ret        real*8 (nt_out)
-  !                    Impurity densities (permanently) retained at the wall over time [1/cm].
   !     N_wall       real*8 (nt_out)
-  !                    Impurity densities (temporarily) at the wall over time [1/cm].
+  !                    Impurity densities (permanently) at the wall over time [1/cm].
```

Comment:
@antonellozito thanks for taking care to correct some of the typos and wrong docs!